tests/validation/tests/v3_api/test_service_discovery.py

```diff
@@ -118,6 +121,10 @@ def test_service_discovery_when_workload_scale_down():
     validate_service_discovery(workload, scale)
 
 
+# The base image of Windows needs to match the host kernel(1809, 1903, ...),
```

Comment:
Does it mean you need to have a multi-arch image for windows `1809`, `1903` or later windows versions?

Reply:
Add `TEST_UPGRADE_IMAGE_1` and `TEST_UPGRADE_IMAGE_2` to test these cases which allow upgrading image.